crates/runtime/tests/models/search.rs

```diff
@@ -976,7 +976,7 @@ async fn test_text_search_multiple_columns() -> Result<(), anyhow::Error> {
             ),
             SearchTestCase::new(
                 "multi_text_column_sql_text_search_basic_question",
-                SearchTestType::Sql("SELECT id, answer, trunc(score, 3) FROM text_search(qs, 'second', question) order by score desc LIMIT 4"),
+                SearchTestType::Sql("SELECT id, question, trunc(score, 3) FROM text_search(qs, 'angles', question) order by score desc LIMIT 4"),
```

Comment:
This was a miss in a previous PR. Updated to test the UDTF for the second index column. 

[crates/runtime/tests/models/snapshots/integration_models__models__search__multi_text_column_sql_text_search_basic_question.snap](https://github.com/spiceai/spiceai/pull/6985/files#diff-c606da4a9ae21ae4a2d7f9fb20b786fa4c3698e6804bdd083df1c96330591000) updated accordingly